docs/reference/analysis/tokenizers/pathhierarchy-tokenizer.asciidoc

```diff
@@ -3,7 +3,9 @@
 
 The `path_hierarchy` tokenizer takes a hierarchical value like a filesystem
 path, splits on the path separator, and emits a term for each component in the
-tree.
+tree. `path_hierarcy` tokenizer uses Lucene's
+https://lucene.apache.org/core/8_0_0/analyzers-common/org/apache/lucene/analysis/path/PathHierarchyTokenizer.html[PathHierarchyTokenizer]
```

Comment:
Not a big deal, but the `{lucene_version_path}` attribute ensures we use the appropriate Lucene version. 
```suggestion
https://lucene.apache.org/core/{lucene_version_path}/analyzers-common/org/apache/lucene/analysis/path/PathHierarchyTokenizer.html[PathHierarchyTokenizer]
```